Bio/SeqUtils/IsoelectricPoint.py

```diff
@@ -25,8 +25,15 @@
 positive_pKs = {"Nterm": 7.5, "K": 10.0, "R": 12.0, "H": 5.98}
 negative_pKs = {"Cterm": 3.55, "D": 4.05, "E": 4.45, "C": 9.0, "Y": 10.0}
 pKcterminal = {"D": 4.55, "E": 4.75}
-pKnterminal = {"A": 7.59, "M": 7.0, "S": 6.93, "P": 8.36, "T": 6.82, "V": 7.44,
-               "E": 7.7}
+pKnterminal = {
+    "A": 7.59,
+    "M": 7.0,
+    "S": 6.93,
+    "P": 8.36,
+    "T": 6.82,
+    "V": 7.44,
+    "E": 7.7,
+}
```

Comment:
How about 
```python
pKnterminal = {
    "A": 7.59, "M": 7.0, "S": 6.93, "P": 8.36, "T": 6.82, "V": 7.44, "E": 7.7
}
```

Reply:
If we want to disable black in this region, that looks fine. But if that is the only case in this file, I would just let black do what it wants.

Reply:
Does this require disabling black? I thought that would be black compatible?

Reply:
This would need disabling black, since is a list of 7 items, short,  I will leave black take care of it.

Reply:
From what I read [here](https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines) I would think that my suggestion would be black compatible. It's something like the second rule: if it doesn't fit in the same line with the opening bracket, try it in an extra line. So, did you do this manually, or was it done by black? If it was done by black, I would file a ticket to them.

Reply:
black does this, It doesn't take your suggestion, it puts one item per line, not even with an extra line.

Reply:
Okay, thank you. Actually, I don't understand the difference in the black examples between a function call with parameters and a function definition with arguments, and why you could do this with a list but not with a dic, but that's not our problem. So, OK.